schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+  id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] [OPTIONAL] Kindly it will be better 😃 if you make the id of the table not null to make the id column not empty in the database and with that your schema will be validated and professional 💯🚀

```id
    id int NOT NULL IDENTITY(1,1)

```